modules/battery_manager/test/TestPlan.md

```diff
@@ -0,0 +1,85 @@
+## Battery Manager Module Test Plan
+
+### 📄 Overview
+
+Battery Manager module allows users to search for, add, activate, and deactivate entries in the
+Test Battery
+
+### 🔒 Permissions 
+
+In order to use the Battery Manager module the user might need one or both of the following permissions:
+
+1. **battery_manager_view** - gives user read-only access to Battery Manager module
+(browse entries in Test Battey)
+2. **battery_manager_edit** - gives user edit access to Battery Manager module
+(add/activate/deactivate entries in Test Battery)
+
+>**Note**: superusers have both of the aforementioned permissions by default! 💪
+
+### 💯 Features
+
+1. **Browse** entries in the Test Battery
+2. **Add** new entry to the `test_battery` table
+  - Instrument, Minimum age, Maximum age, and Stage are required fields
+3. **Activate** entry in the `test_battery` table
+4. **Deactivate** entry in the `test_battery` table
+
+---
+
+### 💻 Testing Procedure
+
+**Install Module** 
+  1. Run associated SQL patches to update permissions and LorisMenu tables.
+
+**Testing with no permissions** [Automation Testing]
+  1. Access the module with a regular user (without superuser permissions)
+  2. By default, the access to module should be denied
+
+**Testing with view permission** [Automation Testing]
+  1. Add view permission to the aforementioned user
+  2. Battery Manager module should be accessible and only present with **one** tab (Browse)
+  3. The **Deactivate** column should not be in the data table
+
+**Testing with edit permission** [Automation Testing]
+  1. Add edit permission
+  2. Battery Manager module should now have **two** tabs (Browse) and (Add)
+  3. The **Deactivate** should be in the data table
+  4. Clicking on Add tab should hide the data table and display a form with the following fields:
+     Instrument, Minimum age, Maximum age, Stage, Subproject, Visit Label, Site, First Visit,
```

Comment:
quotes

Reply:
could you elaborate where I should add quotes?

Reply:
each field as they are label that are shown on the user interface.